packages/databases-collections-list/src/namespace-card.tsx

```diff
@@ -0,0 +1,296 @@
+};
+
+const cardBadge = css({
+  gap: spacing[1],
+});
+
+const cardBadgeLabel = css({});
+
+export type BadgeProp = {
+  name: string;
+  variant?: BadgeVariant;
+  icon?: IconGlyph;
+  hint?: React.ReactNode;
+};
+
+const CardBadge: React.FunctionComponent<BadgeProp> = ({
+  name,
+  icon,
+  variant,
+  hint,
+}) => {
+  const badge = (
+    <Badge className={cardBadge} variant={variant}>
+      {icon && <Icon size="small" glyph={icon}></Icon>}
+      <span className={cardBadgeLabel}>{name}</span>
+    </Badge>
+  );
+
+  if (hint) {
+    return <Tooltip trigger={badge}>{hint}</Tooltip>;
+  }
+
+  return badge;
+};
+
+const card = css({
+  padding: spacing[3],
+});
+
+export type DataProp = {
+  label: React.ReactNode;
+  value: React.ReactNode;
+  hint?: React.ReactNode;
+};
+
+export type NamespaceItemCardProps = {
+  id: string;
+  type: ItemType;
+  viewType: ViewType;
+  name: string;
+  status: string;
+  data: DataProp[];
+  badges?: BadgeProp[] | null;
+  onItemClick(id: string): void;
+  onItemDeleteClick?: (id: string) => void;
+};
+
+const namespaceDataGroup = css({
+  display: 'flex',
+  gap: spacing[2],
+  marginTop: spacing[3],
+});
+
+const column = css({
+  flexDirection: 'column',
+});
+
+export const NamespaceItemCard: React.FunctionComponent<
+  NamespaceItemCardProps &
+    Omit<
+      React.HTMLProps<HTMLDivElement>,
+      Extract<keyof NamespaceItemCardProps, string>
+    >
+> = ({
+  id,
+  type,
+  name,
+  status,
+  data,
+  onItemClick,
+  onItemDeleteClick,
+  badges = null,
+  viewType,
+  ...props
+}) => {
+  const [hoverProps, isHovered] = useHoverState();
+  const [focusProps, focusState] = useFocusState();
+
+  const onDefaultAction = useCallback(() => {
+    onItemClick(id);
+  }, [onItemClick, id]);
+
+  const defaultActionProps = useDefaultAction(onDefaultAction);
+
+  const onDeleteClick = useCallback(
+    (evt) => {
+      evt.stopPropagation();
+      if (onItemDeleteClick) {
+        onItemDeleteClick(id);
+      }
+    },
+    [onItemDeleteClick, id]
+  );
+
+  const badgesGroup = badges && (
+    <CardBadges>
+      {badges.map((badge) => {
+        return <CardBadge key={badge.name} {...badge}></CardBadge>;
+      })}
+    </CardBadges>
+  );
+
+  const cardProps = mergeProps(
+    { className: card },
+    defaultActionProps,
+    hoverProps,
+    focusProps,
+    props
+  );
+
+  const isButtonVisible =
+    onItemDeleteClick &&
+    ([FocusState.FocusVisible, FocusState.FocusWithinVisible].includes(
+      focusState
+    ) ||
+      isHovered);
+
+  return (
+    // @ts-expect-error the error here is caused by passing children to Card
+    // component, even though it's allowed on the implementation level the types
+    // are super confused and don't allow that
+    <Card
+      key={id}
+      contentStyle="clickable"
+      data-testid={`${type}-grid-item-${id}`}
+      {...cardProps}
+    >
+      <CardTitleGroup>
+        <CardName>{name}</CardName>
```

Comment:
Something I think we lose having the name and the whole card clickable is the ability users probably want is the ability to select by clicking and hover a database/collection name. I sometimes use this to copy paste a database name into my code. Is this a behavior we want to move away from?

Reply:
v example copy select

https://user-images.githubusercontent.com/1791149/146314334-35ab9c81-9424-4144-9de9-afc509374d53.mp4

Reply:
I don't think we considered this use-case, I'll ask Claudia

Reply:
We decided to add it as an additional option to the card actions: tracking text select vs card click will be tricky and this in general fits better with an idea of having context menus eventually to make Compass feel more app-like

Reply:
Sounds good, might also want to investigate how easy it will be to add right click actions, we could have copying the name be a right click menu option as well.